clang/include/clang/AST/DeclCXX.h

```diff
@@ -1960,24 +1960,44 @@ class ExplicitSpecifier {
 class CXXDeductionGuideDecl : public FunctionDecl {
   void anchor() override;
 
+public:
+  // Represents the relationship between this deduction guide and the
+  // deduction guide that it was generated from (or lack thereof).
+  // See the SourceDeductionGuide member for more details.
+  enum class SourceDeductionGuideKind : unsigned char {
```

Comment:
```suggestion
  enum class SourceDeductionGuideKind : uint8_t {
```

I thought I saw elsewhere we do this reasonably consistently?  Else, feel free to ignore.